js/currency-converter.js

```diff
@@ -0,0 +1,1027 @@
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "20th"
+    },
+    "1985": {
+        "inflation": 2.01,
+        "horse_price": 2387.37,
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "20th"
+    },
+    "1990": {
+        "inflation": 1.51,
+        "horse_price": 2387.37,
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "20th"
+    },
+    "1995": {
+        "inflation": 1.28,
+        "horse_price": 2387.37,
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "20th"
+    },
+    "2000": {
+        "inflation": 1.12,
+        "horse_price": 2387.37,
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "21st"
+    },
+    "2005": {
+        "inflation": 1,
+        "horse_price": 2387.37,
+        "cow_price": 392.79,
+        "wool_price": 5.36,
+        "wheat_price": 17.36,
+        "wage_price": 57.6,
+        "century": "21st"
+    },
+    "2017": {
+        "inflation": 1.37,
+        "horse_price": 2387.37,
+        "cow_price": 1200,
+        "wool_price": 6.35,
+        "wage_price": 100,
+        "wheat_price": 1.88,
+        "century": "21st"
+    },
+    "century_intros": {
+        "13th": "Most peasants in the thirteenth century lived below a reasonable subsistence level and this standard declined during the century.",
+        "14th": "Between 1315 and 1322, the Great Famine, brought about by unusually prolonged periods of heavy rainfall and cold spells, killed 10-15% of the population.",
+        "15th": "A hierarchy of consumption patterns mirrored the social hierarchy. The poor bought locally and in small quantities, while the better-off ventured to small towns and provincial centres.",
+        "16th": "The sixteenth century was a period of population rise and price inflation. The social pressure on those with wealth to display it was considerable.",
+        "17th": "In the middle of the seventeenth century, the period of price and population rises ended and the country entered a period of stability in both, that was to last until the mid-eighteenth century.",
+        "18th": "One of the immediate and most important effects of the Act of Union in 1707 was the creation of a united free trade area managed from London. ",
+        "19th": "The period from the late 18th century to the mid-Victorian years witnessed a major shake up and change in both the economy and society.",
+        "20th": "The twentieth century saw the most rapid and dramatic social and economic change of any century."
+    }
+};
+
+$(function () {
+    hide_inputs_by_year();
+});
+
+function hide_inputs_by_year() {
```

Comment:
It would be better to apply a class to the inputs that indicate which period they are relevant to and show/hide in response to that. 

```javascript
if(get_currency_year() > 1970) {
   $('.newer-currencies').show();
   $('.older-currencies').hide();
}
```